doc/introduction/index.rst

```diff
@@ -3,383 +3,363 @@
-one of these predefined configurations and customize it to their own
-application scenario needs.
-
-.. list-table:: Scenario-based Best Known Configurations
-   :header-rows: 1
-
-   * - Predefined BKC
-     - Usage Scenario
-     - VM0
-     - VM1
-     - VM2
-     - VM3
-
-   * - Software Defined Cockpit
-     - SDC
-     - Service VM
-     - Post-launched VM
-     - One Kata Containers VM
-     -
-
-   * - Industry Usage Config
-     - Industry
-     - Service VM
-     - Up to 5 Post-launched VMs
-     - One Kata Containers VM
-     - Post-launched RTVM (Soft or Hard real-time)
-
-   * - Hybrid Usage Config
-     - Hybrid
-     - Pre-launched VM (Safety VM)
-     - Service VM
-     - Post-launched VM
-     -
-
-   * - Hybrid real-time Usage Config
-     - Hybrid RT
-     - Pre-launched VM (real-time VM)
-     - Service VM
-     - Post-launched VM
-     -
-
-   * - Logical Partition
-     - Logical Partition
-     - Pre-launched VM (Safety VM)
-     - Pre-launched VM (QM Linux VM)
-     -
-     -
-
-Here are block diagrams for each of these four scenarios.
-
-SDC Scenario
-============
-
-In this SDC scenario, an instrument cluster (IC) system runs with the
-Service VM and an in-vehicle infotainment (IVI) system runs in a user
-VM.
-
-.. figure:: images/ACRN-V2-SDC-scenario.png
-   :width: 600px
-   :align: center
-   :name: ACRN-SDC
+.. _XEN Project:
+   https://wiki.xenproject.org/wiki/Understanding_the_Virtualization_Spectrum
 
-   SDC scenario with two VMs
+There's a price to pay for sharing devices. Whether device emulation is
+performed in the hypervisor, or in user space within an independent VM, overhead
+exists.  This overhead is worthwhile as long as the devices need to be shared by
+multiple guest operating systems. If sharing is not necessary, then there are
+more efficient methods for accessing devices, for example "passthrough".
 
-Industry Scenario
-=================
+The ACRN device model is a placeholder of the User VM. It allocates memory for
+the User OS, configures and initializes the devices used by the User VM, loads
+the virtual firmware, initializes the virtual CPU state, and invokes the ACRN
+hypervisor service to execute the guest instructions.  ACRN Device model is an
+application running in the Service VM that emulates devices based on command
+line configuration
 
-In this Industry scenario, the Service VM provides device sharing capability for
-a Windows-based HMI User VM. One post-launched User VM can run a Kata Container
-application. Another User VM supports either hard or soft real-time OS
-applications. Up to five additional post-launched User VMs support functions
-such as human/machine interface (HMI), artificial intelligence (AI), computer
-vision, etc.
+See the :ref:`hld-devicemodel` developer reference for more information.
 
-.. figure:: images/ACRN-Industry.png
-   :width: 600px
-   :align: center
-   :name: Industry
+Device Passthrough
```

Comment:
We use the ter `guest` a lot in this paragraph. Shouldn't we use `User VM` or `VM` instead?